shared/src/main/scala/amf/core/internal/metamodel/document/BaseUnitModel.scala

```diff
@@ -28,6 +28,13 @@ trait BaseUnitModel extends ModelDefaultBuilder {
                                        "location",
                                        "Location of the metadata document that generated this base unit"))
 
+  val Package: Field = Field(Str,
```

Comment:
Why not call it Namespace/Module instead of Package? Namespace sounds more general. Package sounds to Java/Grpc oriented. (Module isn't great as we have an entity called like that.)

Reply:
Because is not. Grpc defined packages and can import packages from other files